game/static/game/css/cursor.css

```
@@ -0,0 +1,192 @@
+.cursor {
+    position: fixed;
+    pointer-events: none;
+
+    z-index: 9990;
```

Stylelint (17.14.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)